azure-kusto-ingest/source/columnMappings.ts

```diff
@@ -38,6 +38,10 @@ interface ApiColumnMapping {
     Properties?: MappingPropertiesStrings;
 }
 
+interface HasToString {
```

Comment:
I don't understand this interface... all objects has "toString()"

Reply:
I wanted a type that is "all non-null values", this type in typescript is "{}", but the linter recommends against it because it could be misused.

But you're right, so I changed the type to {} and disabled the lint with an explanation.